guides/tutorials/issue-a-credential.md

```diff
@@ -211,17 +259,12 @@ import { CredentialProtocolVersion, CredentialPreviewAttribute } form '@aries-fr
 const credentialExchangeRecord = agent.credentials.offerCredential({
   connectionId,
   protocolVersion: CredentialProtocolVersion.V1,
+  credentialDefinitionId: credentialDefinition.id,
```

Comment:
credentialDefinitionId is part of the `indy` key

Reply:
Snippets will be fixed in the next pr.